src/components/steps/step-sheet/DetailsTab.tsx

```diff
@@ -36,6 +46,32 @@ export function StepDetailsTab({ stepId, runId }: Props) {
 	const enable_artifact_metadata = data.metadata?.config?.enable_artifact_metadata;
 	const enable_artifact_visualization = data.metadata?.config?.enable_artifact_visualization;
 
+	const urlCodeRepo = () => {
```

Comment:
A more descriptive name could be something like `getRepositoryLink` or whatever, but its just a recommendation